packages/engine/tests/units/multiple_groups/sugarscape/README.md

```diff
@@ -0,0 +1,23 @@
+# Antiquities Act
```

Comment:
This file does not belong here

Reply:
Where does this keep coming from 😂

Reply:
It strikes again! Where is it coming from?